src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2369,6 +2387,12 @@ theorem getLsbD_twoPow (i j : Nat) : (twoPow w i).getLsbD j = ((i < w) && (i = j
           simp at hi
         simp_all
 
+@[simp]
+theorem getElem_twoPow {i j : Nat} (h : j < w) : (twoPow w i)[j] = decide (i = j) := by
```

Comment:
This is a bit dumb, but could we have `decide (j = i)` instead? `j` feels more like "the variable" here.

Similarly for `getElem_one` below.

Reply:
Done.